grid/chat/workspaces/profile.supr

```diff
@@ -0,0 +1,170 @@
+"""
+Chat Workspaces
+Retrieve workspaces for the user.
+"""
+name = "chat/workspaces"
+version = "1.0.0"
+
+"""
+GetWorkspaces usecase
+"""
+usecase GetWorkspaces {
+  input {
+    """
+    As Bot
+    Boolean representing whether to authorize as bot
```

Comment:
I don't find this explanation very helpful. Also it's not clear to me why this is needed here since all other maps authorize the user as bot...

Reply:
My understanding of this field is limited, so pick this suggestion with grain of salt. My issues with this comment:

- the fact that it's boolean is redundant,
- it's not clear what the default behavior is,
- it's not clear what's the impact.

```suggestion
    Whether the access token authorizes a user or bot. By default it authorizes a user.
```

Reply:
@jnv I'm not even sure why I added this field when we use bot auth everywhere. Probably to start discussion whether to include it in other usecases as well. 🤔

Reply:
Do you plan to remove this field? Really, this should be an integration parameter.